canonical/snapcraft.io
# Add models table

## Done
Added the models table and filter using dummy data

## How to QA
- Go to /admin/<STORE_ID>/models
- Check that there is a table of models
- Check that the models table is sortable by "Name", "Last updated" and "Created date"
- Check that the "Search models" filter works
- Check that reloading the page with a filter that it is applied

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-4734
